.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,57 @@
+name: genesis-verify
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "misc/deployments/**/genesis.json"
```

Comment:
Can you make this workflow take in a matrix, so we can specify in the workflow which testnets are relevant for verification?

Not all genesis files can be verified correctly with the latest version of `gnogenesis`

Reply:
9f3ec963edd74af104fe7e511d9355434763c3dd